frontend/src/pages/modelServing/screens/metrics/ScheduledMetricSelect.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Select, SelectOption } from '@patternfly/react-core';
+
+type ScheduledMetricSelectProps = {
+  selected: string | undefined;
+  options: string[];
+  onSelect: (name: string) => void;
+};
+const ScheduledMetricSelect: React.FC<ScheduledMetricSelectProps> = ({
+  selected,
+  options,
+  onSelect,
+}) => {
+  const [isOpen, setOpen] = React.useState(false);
+
+  const onToggle = () => setOpen(!isOpen);
+  const onSelectValue = (event, selection) => {
+    onSelect(selection);
+    setOpen(false);
+  };
+
+  return (
+    <Select onToggle={onToggle} isOpen={isOpen} onSelect={onSelectValue} selections={selected}>
+      {options.map((value, index) => (
+        <SelectOption key={index} value={value} />
```

Comment:
try to avoid using index as a key

Reply:
Can do, I'll double check if value is guaranteed unique, I'm not 100% sure it is, I'll look for a more deterministic alternative if not.